malib/settings.py

```diff
@@ -21,8 +21,9 @@
 DEFAULT_EPISODE_CAPACITY = 30000  # int(1e15)
 # related to each group of expr settings
 DEFAULT_EPISODE_BLOCK_SIZE = int(75)
-PICKLE_PROTOCOL_VER = 5
+PICKLE_PROTOCOL_VER = 4
```

Comment:
why downgrade the pickle version?

Reply:
From my test, it seems that Protocol Version 5 can not be used in Python 3.7.10.